Bottom: b325ccfd975dd153285cfe5ba9f66efd16427780
Top:    3513087cec3ea33b2826cc40e6f7fcdcafbab00c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-13 14:48:06 -0700

Refresh of fork-connect

---

diff --git a/src/preload.c b/src/preload.c
index 2b115d3..76c473b 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -574,7 +574,7 @@ static void fork_passive(int socket)
 		goto lclose;
 
 	msg = 0;
-	len = real.write(sfd, &msg, sizeof msgs);
+	len = real.write(sfd, &msg, sizeof msg);
 	if (len != sizeof msg)
 		goto lclose;
 
@@ -622,7 +622,6 @@ static inline enum fd_type fd_fork_get(int index, int *fd)
 int connect(int socket, const struct sockaddr *addr, socklen_t addrlen)
 {
 	int fd, ret;
-	long flags;
 
 	if (fd_get(socket, &fd) == fd_rsocket) {
 		ret = rconnect(fd, addr, addrlen);
